torch/utils/_cxx_pytree.py

```diff
@@ -475,6 +476,11 @@ def map_only(__type_or_types: Type2[T, S]) -> MapOnlyFn[Fn2[T, S, Any]]:
     ...
 
 
+@overload
+def map_only(__type_or_types: Type3[T, S, U]) -> MapOnlyFn[Fn3[T, S, U, Any]]:
```

Comment:
For my understanding: what is `Type3[T, S, U]`?

Reply:
It's a 3-tuple of types. For example:

```python
isinstace(x, (int, float, torch.Tensor))
             ^------------------------- Type3[int, float, torch.Tensor]
```